docs/Documentation/Development/VSCode/index.md

```diff
@@ -77,3 +77,33 @@ Now use the Remote-SSH extension in VS Code to `Connect to Host...` and use the
 
 This should open a new VS Code window that will connect to the compute node automatically. You may begin browsing your home directory and editing files in the VS Code window.
 
+## Jupyter in VS Code on Kestrel
+
+### Setting Up VS Code
+To begin, proceed to VSCode and install these additional extensions, if you do not already have them: [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python) and [Jupyter](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter).
+
+### Setting Up Conda Environment
+In addition, you will also need to set up a Python Environment on Kestrel. At a minimum, you must perform the following commands:
+
+```
+module load anaconda3
+conda create -n myJupEnv
+conda activate myJupEnv
+conda install Jupyter
+```
+
+### Intializing Environment
+
+Before using VS Code to connect to a compute, the compute node has to be allocated. Log onto Kestrel via a terminal and allocate 2 nodes by using this line:
```

Comment:
This command isn't allocating two nodes

Reply:
Removed that section as it was redundant and explained earlier in the doc. See ab74f8f